specification/compute/resource-manager/Microsoft.Compute/stable/2018-10-01/compute.json

```diff
@@ -7801,10 +7801,6 @@
           "type": "boolean",
           "description": "Specifies whether the latest model has been applied to the virtual machine."
         },
-        "protectFromScaleIn" : {
-          "type": "boolean",
-          "description": "Specifies whether the virtual machine instance shouldn't be considered for deletion during a scale-in operation"
-        },
```

Comment:
Did we follow the ARM guideline for deprecating properties ?

Reply:
Yes. This must be done in new api version. Refer to 'versioning/deprecation' section under 'API contracts' of the ARM wiki. http://aka.ms/armwiki

Reply:
This should be nested inside another property. It's not usable currently as-is.